src/components/actionButtion/actionButton.html

```diff
@@ -0,0 +1,8 @@
+<button class="btn">
+  <img
+    src="../../assets/btn-plus.svg"
+    alt="add new text"
+    height="40px"
+    width="40px"
```

Comment:
Good call! However, if you set `height` and `width` like this, you don't need the `px` unit. This will be interpreted for from the browser and is not the same as `height`/`width` in your CSS styles. With this you tell the browser that this would be the size and he should add a placeholder if the image doesn't load. Google: layout shifts in CSS
```suggestion
    height="40"
    width="40"
```